src/streaming/metrics/utils/ManifestParsing.js

```diff
@@ -88,10 +88,16 @@ function ManifestParsing (config) {
                             return;
                         }
 
-                        for (const prop in reporting) {
-                            if (reporting.hasOwnProperty(prop)) {
-                                reportingEntry[prop] = reporting[prop];
-                            }
+                        if (reporting.hasOwnProperty('value')) {
+                            reportingEntry.value = reporting.value;
+                        }
+
+                        if (reporting.hasOwnProperty(constants.DVB_REPORTING_URL)) {
+                            reportingEntry.dvb_reportingUrl = reporting[constants.DVB_REPORTING_URL];
```

Comment:
Can we stick to camel case here? dvbReportingUrl

Reply:
Similar to the previous comment, matched the style for [other dvb attributes](https://github.com/Dash-Industry-Forum/dash.js/blob/4961649e62505bd16e9aabb3acc1f689577ba048/src/dash/vo/BaseURL.js#L45).